### PR TITLE
add cloudwatch:GetMetricData STS installer permission policy

### PR DESCRIPTION
**Pull request by zvikorn**
Signed-off-by: Tzvika Avni <tavni@redhat.com>

[SDA-5791](https://issues.redhat.com/browse/SDA-5791)

@pvasant

### CODE REVIEW

File: resources/sts/4.10/sts_installer_permission_policy.json [hunk 1]
```diff
@@ -182,7 +182,8 @@
                 "ec2:DescribeVpcEndpointServicePermissions",
                 "ec2:DescribeVpcEndpointServices",
                 "ec2:ModifyVpcEndpointServicePermissions",
-                "kms:DescribeKey"
+                "kms:DescribeKey",
+                "cloudwatch:GetMetricData"
```
Comment (fahlmant) [inline resources/sts/4.10/sts_installer_permission_policy.json:186]: Whats the justification for this metric? Also does this need to be added to the 4.9 installer role as well?

Comment (zvikorn) [reply resources/sts/4.10/sts_installer_permission_policy.json:186]: https://coreos.slack.com/archives/C03B98SL8L8/p1649345512136449
Getting vCPU's usage for preflight check:
https://issues.redhat.com/browse/SDA-5138

Comment (zvikorn) [reply resources/sts/4.10/sts_installer_permission_policy.json:186]: @pvasant can answer your 4.8/4.9 question

Comment (fahlmant) [reply resources/sts/4.10/sts_installer_permission_policy.json:186]: That slack link isn't working for me. Can you please verify it?

SDA-5791 has none of this information. Can you please update it to reflect the justification?

Comment (zvikorn) [reply resources/sts/4.10/sts_installer_permission_policy.json:186]: done

Comment (pvasant) [reply resources/sts/4.10/sts_installer_permission_policy.json:186]: @zvikorn we need to add it to all the versions.

**Comment by pvasant** [comment]
@cblecker @fahlmant  can we get this merged

**Comment by fahlmant** [comment]
I'm not sure if this permission is approved.

**Comment by pvasant** [comment]
@arendej  approved this permission and this is for our preflight checks

**Comment by fahlmant** [comment]
Can you link to the approval?

**Review by fahlmant — APPROVED** [review]
/appove

**Review by jharrington22 — APPROVED** [review]

**Pull request merged**